pulp_rpm/app/tasks/copy.py

```diff
@@ -82,7 +82,8 @@ def copy_content(config, dependency_solving):
         source_repo_version = RepositoryVersion.objects.get(pk=entry["source_repo_version"])
         dest_repo = RpmRepository.objects.get(pk=entry["dest_repo"])
 
-        if entry.get("dest_base_version"):
+        dest_version_provided = bool(entry.get("dest_base_version"))
```

Comment:
@dralley, is the version a string and not an int? e.g. `"0"`.
I'm just confused why this approach helped.

Reply:
The ```bool()``` isn't necessary, I just wanted to make the type correct rather than passing around an object / None.

```dest_repo_version``` is overwritten on line 89 so it can't be provide this information afterwards.

Reply:
I got confused by the `dest_repo_version` from the viewset.
No concerns or questions here.